server/odc-service/src/main/java/com/oceanbase/odc/service/connection/util/ConnectionInfoUtil.java

```diff
@@ -61,6 +64,11 @@ public static void initConnectionId(@NonNull Statement statement, @NonNull Conne
             String sessionId = queryConnectionId(statement, connectionSession.getDialectType());
             Verify.notNull(sessionId, "SessionId");
             connectionSession.setAttribute(ConnectionSessionConstants.CONNECTION_ID_KEY, sessionId);
+            if (connectionSession.getDialectType().isOceanbase() && VersionUtils.isGreaterThanOrEqualsTo(
+                    ConnectionSessionUtil.getVersion(connectionSession), "4.2")) {
+                String proxySessId = queryProxySessId(statement, connectionSession.getDialectType(), sessionId);
+                connectionSession.setAttribute(ConnectionSessionConstants.CONNECTION_PROXY_SESSID_KEY, proxySessId);
```

Comment:
we may use another special key to identify this session id, may `OB_PROXY_SESSID_KEY`?

Reply:
sure , I renamed it to `OB_PROXY_SESSID_KEY`